app/src/main/java/com/svs/hztb/Activities/MobilePhoneRegistrationActivity.java

```diff
@@ -0,0 +1,39 @@
+package com.svs.hztb.Activities;
+
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.view.Menu;
+import android.view.MenuItem;
+
+import com.svs.hztb.R;
+
+public class MobilePhoneRegistrationActivity extends AppCompatActivity {
```

Comment:
change the name of the class to RegistrationActivity. MobilePhone is reduntant because we have only one way of regstration